### PR TITLE
Update ParallelProj minimum version

### PR DESCRIPTION
**Pull request by samdporter**
v1.2.13 has bug fix in https://github.com/gschramm/parallelproj/blob/master/c/include/parallelproj_c.h \\
Bug causes error in STIR compilation.

### CODE REVIEW

**Comment by samdporter** [comment]
I guess this parallelproj version is very new. I'm a bit confused as to why this problem hasn't previously occurred

**Comment by KrisThielemans** [comment]
we were using 0.8 before. Only recently switched to 1.x.

I'm surprised that it gives problems on `master` as I didn't think the tof stuff would be included.

**Comment by samdporter** [comment]
I've double checked the cmake options for my SIRF build and STIR_TAG is definitely origin/master

**Comment by KrisThielemans** [comment]
currently failing as we test with
https://github.com/UCL/STIR/blob/593c06909201419716398d699211614eedd93df1/.github/workflows/build-test.yml#L151

**Comment by gschramm** [comment]
Hi Sam, hi Kris,

1. The errors I see in the build logs, seem more like azure connection errors to me. Or did I miss an error before that?
2. The ";" vs "," bug in the header file, was indeed fixed in v1.0.1
3. the API of v1.2.13 should be backward compatible with v1.0.1. I try to stick to semantic version numbers, meaning that if  the API changes, I increase the major version number. After the latest discussions with @markus-jehl, the build with v1.2.13 should be fine.
4. You don't need ```git clone``` anymore to build ```parallelproj``` with the correct version number, since the version number is now read from [a file](https://github.com/gschramm/parallelproj/blob/master/package.json) that gets auto-bumped by ```ver-bump``` before I release (tag) a new version.

**Comment by KrisThielemans** [comment]
@samdporter please change to require 1.0.1. It might be that CMake ignores the patch, but at least it's clearer for the developer what we need.

File: CMakeLists.txt [hunk 1]
```diff
@@ -197,16 +197,16 @@ endif()
 
 # Parallelproj
 if(NOT DISABLE_Parallelproj_PROJECTOR)
-    find_package(parallelproj 1.0 CONFIG)
+    find_package(parallelproj CONFIG)
```
Comment (KrisThielemans) [inline CMakeLists.txt:200]: I guess still best to keep 1.0 here. why delete it? If anything, I'd put 1.0.1 here.

**Review by KrisThielemans — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (samdporter) [reply CMakeLists.txt]: A misread of CMake find_package documentation. Fixed (I think)

**Pull request merged**